great_expectations/expectations/core/schemas/ExpectColumnPairValuesAToBeGreaterThanB.json

```diff
@@ -80,12 +80,12 @@
         },
         "mostly": {
             "title": "Mostly",
-            "default": 1.0,
```

Comment:
Lotta noise on these. It's functionally the same, just a different ordering for `mostly`.